libraries/AP_CANManager/AP_SLCANIface.cpp

```diff
@@ -685,7 +685,13 @@ int16_t SLCAN::CANIface::receive(AP_HAL::CANFrame& out_frame, uint64_t& rx_time,
         // Also send this frame over can_iface when in passthrough mode,
         // We just push this frame without caring for priority etc
         if (_can_iface) {
-            _can_iface->send(out_frame, AP_HAL::native_micros64() + 1000, out_flags);
+            if (_can_iface->send(out_frame, AP_HAL::native_micros64() + 1000, out_flags)) {
+                rx_queue_.pop();
```

Comment:
should we timeout packets after 8 tries like we do in AP_Periph?

Reply:
yeah, that might be a good idea.